pkg/synthetictests/image_pulls.go

```diff
@@ -96,5 +96,14 @@ func newSingleEventCheckRegex(testName, regex string) *singleEventCheckRegex {
 //
 func testBackoffPullingRegistryRedhatImage(events monitorapi.Intervals) []*junitapi.JUnitTestCase {
 	testName := "[sig-arch] should not see excessive pull back-off on registry.redhat.io"
-	return newSingleEventCheckRegex(testName, imagePullRedhatRegEx).test(events)
+	return newSingleEventCheckRegex(testName, imagePullRedhatRegEx, math.MaxInt, imagePullRedhatFlakeThreshold).test(events)
+}
+
+// testRequiredInstallerResourcesMissing looks for this symptom:
```

Comment:
Looks like you meant to paste the symptom here.

Reply:
you're right.  Fixed